src/results.md

```diff
@@ -4,34 +4,49 @@ title: Results
 permalink: /results/
 include: true
 ---
-Having the world’s largest database of Syriac letter forms allows one to both describe and quantify important developments in Syriac manuscript culture. In some cases, this results in computer assisted paleography. That is, automated results provide data that help human paleographers better localize Syriac manuscripts. In other cases, our data challenges long held assumptions in the field (e.g. the traditional way of categorizing Syriac script) or opens up new fields of inquiry (e.g. the relationship between script and genre). Although more details appear in project publications, the following provide several quick case examples for how large data sets, digital analysis, and visual analytics can help researchers address key philological and historical problems.
+Having the world’s largest database of Syriac letter forms allows one to both describe and quantify important developments in Syriac manuscript culture. In some cases, this helps with computer assisted paleography. That is, automated results provide data that allow human paleographers to better localize Syriac manuscripts. In other cases, project data challenges long held assumptions in the field (e.g. the traditional way of categorizing Syriac script) or opens up new fields of inquiry (e.g. the relationship between script and genre). Although more details appear in project publications, the following provide several quick case examples for how large data sets, digital analysis, and visual analytics can help researchers address key philological and historical problems. 
 
-## Automatic Script Charts
+The hope is that these visuals will spur ideas for your own research projects. If you find DASH helpful for a presentation or publication, please do contact us. We would love to include a citation, link, or abstract of your work on this website. Knowing that you are using DASH will also help us justify future updates and expansions to the project. In addition to the images that are publically accessible on this website, we have much additional paleographic data that may be helpful to researchers. If you have a project for which you think a larger dataset could be of use, do send us a note using the “contact” menu above.  
```

Comment:
do send us a note using the "contact" menu below. [Assuming that the contact menu will be on the footer, which we are obliged to add as per #67]

Reply:
We should ask Michael. I remember him wanting a full page for it which would go in the normal nav menu. Maybe it goes in both places?

Reply:
Yeah, if I recall the conversation correctly, the idea was for the "Contact" link (wherever it may reside) to point to a form like [this one](https://docs.google.com/forms/d/1OIEZ8jrkfaZChTKHBodNw_IzmkVnTagXB9vBJ1vKLRU/edit).
If we decide to follow the [dictum](https://identity.stanford.edu/web-mobile.html) that all Stanford websites should include the "Stanford Global Footer" then this would seem like a suitable home for the "Contact" link. Given that the footer requirement is roundly ignored by most sites, however, I'm in favor of doing whatever looks good and is easy to find... and that Michael is happy with.

Reply:
Perhaps for now we leave as is then and adjust once we have an implemented contact page.